bazaarvoice/emodb
# Pd-174028 common client module for javax 2.x

## Github Issue #

[PD- 174028](https://bazaarvoice.atlassian.net/browse/PD-174028)

## What Are We Doing Here?

1. Create separate common client module similar to the [existing module](https://github.com/bazaarvoice/emodb/tree/master/common/client/src/main/java/com/bazaarvoice/emodb/client) with javax version 2.x. 

This PR also helps the blocker with Databus client PR

## How to Test and Verify

1. Check out this PR
2. Run Command i. `mvn dependency:tree` and  ii. `mvn clean install` 

## Risk

### Level 

`Low`

### Required Testing

`Smoke`, `Regression`, or `Manual`. (All changes except documentation need smoke
testing at a minimum).
